src/constants.js

```diff
@@ -216,3 +216,13 @@ export const TIMEOUT = 10000;
  * Default timeout for send tokens request in milliseconds
  */
 export const SEND_TOKENS_TIMEOUT = 300000;
+
+/**
+ * Number of iterations to execute when hashing the password
+ */
+export const HASH_ITERATIONS = 1000;
```

Comment:
Include some comments explaining that, even though NIST recommends at least 10,000 iterations, we've tested in real devices, and it took ~3s in iPhone 7 and ~1,5s in Galaxy S8. Thus, we decided to keep 1,000 for now.

Reply:
Done!